src/kokab/one_powerlaw_one_peak/ppd.py

```
@@ -42,12 +42,22 @@ def model(raw: bool, **params) -> DistributionLike:
     validate_args = params.pop("validate_args", True)
     _model = SmoothedPowerlawAndPeak(**params, validate_args=validate_args)
     if raw:
-        _model._component_distributions[0].marginal_distributions[0] = (
-            _model._component_distributions[0].marginal_distributions[0].base_dist
-        )
-        _model._component_distributions[1].marginal_distributions[0] = (
-            _model._component_distributions[1].marginal_distributions[0].base_dist
-        )
+        if isinstance(_model._component_distributions[0], TransformedDistribution):
```

Pylint (3.3.7)

[warning] Access to a protected member _component_distributions of a client class

(W0212)